engine/crates/engine-v2/schema/src/builder/mod.rs

```diff
@@ -103,8 +146,8 @@ impl SchemaBuilder {
                     name: name.into(),
                     arguments: {
                         let map = arguments
-                            .into_iter()
-                            .map(|(id, value)| (id.into(), self.insert_value(value)))
+                            .iter()
```

Comment:
could be `into_iter()` again then I think

Reply:
fixed